tests/fixtures/sleeps-composefile/one-exits.yml

```diff
@@ -0,0 +1,6 @@
+simple:
+  image: busybox:latest
+  command: sleep 200
+another:
+  image: busybox:latest
+  command: sleep 0.5
```

Comment:
I think we could even do `echo something`, that way we're also testing that there isn't any race condition when containers exit immediately (which has been an issue before).

It also means the test doesn't need to wait 0.5 seconds to complete.